### DISCUSSION TITLE
Some mp4 files upload and process while others do not

### DISCUSSION THREAD

**Discussion by dxburge** [mediacms-io/mediacms#1068] [Q&A]
Starting with a fresh MediaCMS single server installation, some of my mp4 files load and process successfully, while others upload and then fail (after retrying 2/2) and then generate an "upload failed" message.
For example, I have a file (A.mp3) size 1.5 GB. This file uploads successfully and I see a link to "View Media"
Another file (B.mp4) is slightly smaller 1.4 GB. This file goes through the upload process up to 100% and then says processing, followed by "retrying 2/2" and then "upload failed"

How can I resolve this?

**Comment by mgogoulos** [comment]
This looks strange, can you try with smaller files so that if there is an error it can be more easily reproduced?
Eg place 10 small (<10MB) videos to a folder and upload them all. Does this end without issues?

one thing you can do is open a `tail -f logs/*` on the terminal while uploading the content. If there is an issue, it should be logged there. Check and add it here in this case..

**Reply by dxburge** [reply]
Out of the 1500+ videos I have, the smallest one that has this issue is roughly 20MB.
I am very new to linux so hopefully I'm giving you here the log info you suggested:

==> logs/debug.log <==
Internal Server Error: /fu/upload/
Traceback (most recent call last):
  File "/home/mediacms.io/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/mediacms.io/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/mediacms.io/lib/python3.10/site-packages/django/views/generic/base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/mediacms.io/mediacms/uploader/views.py", line 49, in dispatch
    return super(FineUploaderView, self).dispatch(request, *args, **kwargs)
  File "/home/mediacms.io/lib/python3.10/site-packages/django/views/generic/base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/mediacms.io/lib/python3.10/site-packages/django/views/generic/edit.py", line 153, in post
    return self.form_valid(form)
  File "/home/mediacms.io/mediacms/uploader/views.py", line 68, in form_valid
    new = Media.objects.create(media_file=myfile, user=self.request.user)
  File "/home/mediacms.io/lib/python3.10/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/mediacms.io/lib/python3.10/site-packages/django/db/models/query.py", line 658, in create
    obj.save(force_insert=True, using=self.db)
  File "/home/mediacms.io/mediacms/files/models.py", line 367, in save
    super(Media, self).save(*args, **kwargs)
  File "/home/mediacms.io/lib/python3.10/site-packages/django/db/models/base.py", line 814, in save
    self.save_base(
  File "/home/mediacms.io/lib/python3.10/site-packages/django/db/models/base.py", line 892, in save_base
    post_save.send(
  File "/home/mediacms.io/lib/python3.10/site-packages/django/dispatch/dispatcher.py", line 176, in send
    return [
  File "/home/mediacms.io/lib/python3.10/site-packages/django/dispatch/dispatcher.py", line 177, in <listcomp>
    (receiver, receiver(signal=self, sender=sender, **named))
  File "/home/mediacms.io/mediacms/files/models.py", line 1373, in media_save
    instance.media_init()
  File "/home/mediacms.io/mediacms/files/models.py", line 430, in media_init
    self.set_media_type()
  File "/home/mediacms.io/mediacms/files/models.py", line 459, in set_media_type
    ret = helpers.media_file_info(self.media_file.path)
  File "/home/mediacms.io/mediacms/files/helpers.py", line 370, in media_file_info
    stream_size = sum([int(line) for line in stdout.split("\n") if line != ""])
  File "/home/mediacms.io/mediacms/files/helpers.py", line 370, in <listcomp>
    stream_size = sum([int(line) for line in stdout.split("\n") if line != ""])
ValueError: invalid literal for int() with base 10: '20600|'

**Reply by MAXimized490** [reply]
I'm having the exact same issue but I'm on Docker and wanted to share what I am seeing in the logs.

The only thing I see is in logs/debug.log where I get the following error twice (for each retry, I assume):
`AttributeError: 'NoneType' object has no attribute 'read'`

Here is the full log:
```
Internal Server Error: /fu/upload/
Traceback (most recent call last):
  File "/home/mediacms.io/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/mediacms.io/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mediacms.io/lib/python3.11/site-packages/django/views/generic/base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mediacms.io/mediacms/uploader/views.py", line 49, in dispatch
    return super(FineUploaderView, self).dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mediacms.io/lib/python3.11/site-packages/django/views/generic/base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mediacms.io/lib/python3.11/site-packages/django/views/generic/edit.py", line 153, in post
    return self.form_valid(form)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/mediacms.io/mediacms/uploader/views.py", line 62, in form_valid
    self.upload.save()
  File "/home/mediacms.io/mediacms/uploader/fineuploader.py", line 113, in save
    chunk = self._save_chunk()
            ^^^^^^^^^^^^^^^^^^
  File "/home/mediacms.io/mediacms/uploader/fineuploader.py", line 109, in _save_chunk
    return self.storage.save(self.chunk_file, self.file)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mediacms.io/lib/python3.11/site-packages/django/core/files/storage/base.py", line 38, in save
    name = self._save(name, content)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mediacms.io/lib/python3.11/site-packages/django/core/files/storage/filesystem.py", line 110, in _save
    for chunk in content.chunks():
  File "/home/mediacms.io/lib/python3.11/site-packages/django/core/files/base.py", line 60, in chunks
    data = self.read(chunk_size)
           ^^^^^^^^^
  File "/home/mediacms.io/lib/python3.11/site-packages/django/core/files/utils.py", line 42, in <lambda>
    read = property(lambda self: self.file.read)
                                 ^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'read'
Internal Server Error: /fu/upload/
Traceback (most recent call last):
  File "/home/mediacms.io/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/mediacms.io/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mediacms.io/lib/python3.11/site-packages/django/views/generic/base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mediacms.io/mediacms/uploader/views.py", line 49, in dispatch
    return super(FineUploaderView, self).dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mediacms.io/lib/python3.11/site-packages/django/views/generic/base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mediacms.io/lib/python3.11/site-packages/django/views/generic/edit.py", line 153, in post
    return self.form_valid(form)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/mediacms.io/mediacms/uploader/views.py", line 62, in form_valid
    self.upload.save()
  File "/home/mediacms.io/mediacms/uploader/fineuploader.py", line 113, in save
    chunk = self._save_chunk()
            ^^^^^^^^^^^^^^^^^^
  File "/home/mediacms.io/mediacms/uploader/fineuploader.py", line 109, in _save_chunk
    return self.storage.save(self.chunk_file, self.file)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mediacms.io/lib/python3.11/site-packages/django/core/files/storage/base.py", line 38, in save
    name = self._save(name, content)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mediacms.io/lib/python3.11/site-packages/django/core/files/storage/filesystem.py", line 110, in _save
    for chunk in content.chunks():
  File "/home/mediacms.io/lib/python3.11/site-packages/django/core/files/base.py", line 60, in chunks
    data = self.read(chunk_size)
           ^^^^^^^^^
  File "/home/mediacms.io/lib/python3.11/site-packages/django/core/files/utils.py", line 42, in <lambda>
    read = property(lambda self: self.file.read)
                                 ^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'read'
```

**Comment by mgogoulos** [comment]
@dxburge these are the logs, so the problem seems to be the same as here: https://github.com/mediacms-io/mediacms/pull/925/files

Can you upload this file in https://demo.mediacms.io/ so that I can verify that the fix works? Then I can merge the PR and this should fix it.

**Reply by dxburge** [reply]
I uploaded the mp4 to the demo site (under user name dbdb) and it progressed all the way to 100% and then failed on processing just as it does on my installation. To clarify, the mp4 is not corrupt and is playable on two different media players that I tried.

**Reply by mgogoulos** [reply]
I've merged the PR, will create a new version now and deploy it on https://demo.mediacms.io/ in order to test it (by triggering to re-transcode the file)

**Reply by mgogoulos** [reply]
the video has succeeded, so the patch was succesfull :)

**Reply by dxburge** [reply]
That's great news! What do I need to do to to upgrade my installation?

**Reply by mgogoulos** [reply]
check out the update section on https://github.com/mediacms-io/mediacms/blob/main/docs/admins_docs.md , there is an update section in both ways (docker + server install)

**Reply by dxburge** [reply]
> check out the update section on https://github.com/mediacms-io/mediacms/blob/main/docs/admins_docs.md , there is an update section in both ways (docker + server install)

I followed your instructions (hopefully correctly), yet these few mp4 files won't upload & process.
These are the steps I've taken:

<pre>root@VMMediaCMS:~# cd /home/mediacms.io/mediacms # enter mediacms directory
root@VMMediaCMS:/home/mediacms.io/mediacms# source  /home/mediacms.io/bin/activate # use virtualenv~
(mediacms.io) root@VMMediaCMS:/home/mediacms.io/mediacms# git pull # update code
fatal: detected dubious ownership in repository at &apos;/home/mediacms.io/mediacms&apos;
To add an exception for this directory, call:

	git config --global --add safe.directory /home/mediacms.io/mediacms
(mediacms.io) root@VMMediaCMS:/home/mediacms.io/mediacms# ^C
(mediacms.io) root@VMMediaCMS:/home/mediacms.io/mediacms# git config --global --add safe.directory /home/mediacms.io/mediacms
(mediacms.io) root@VMMediaCMS:/home/mediacms.io/mediacms# git pull # update code
remote: Enumerating objects: 295, done.
remote: Counting objects: 100% (295/295), done.
remote: Compressing objects: 100% (127/127), done.
remote: Total 295 (delta 178), reused 229 (delta 165), pack-reused 0 (from 0)
Receiving objects: 100% (295/295), 276.64 KiB | 1.98 MiB/s, done.
Resolving deltas: 100% (178/178), completed with 41 local objects.
From https://github.com/mediacms-io/mediacms
   94c646f..ef4067c  main                  -&gt; origin/main
   c2b7c2d..469b689  feat-translations-all -&gt; origin/feat-translations-all
 * [new tag]         v3.5.0                -&gt; v3.5.0
 * [new tag]         v3.4.0                -&gt; v3.4.0
Updating 94c646f..ef4067c
Fast-forward
 cms/permissions.py                                 |  28 <font color="#26A269">+++++</font>
 cms/settings.py                                    |   4 <font color="#26A269">+</font>
 docs/admins_docs.md                                |  24 <font color="#26A269">+++</font><font color="#C01C28">-</font>
 files/helpers.py                                   |   2 <font color="#26A269">+</font><font color="#C01C28">-</font>
 files/views.py                                     |  13 <font color="#26A269">++</font><font color="#C01C28">-</font>
 frontend/packages/player/dist/mediacms-player.css  | 124 <font color="#26A269">++++++++++</font><font color="#C01C28">-----------</font>
 .../vjs-plugin/dist/mediacms-vjs-plugin.css        | 124 <font color="#26A269">++++++++++</font><font color="#C01C28">-----------</font>
 frontend/src/static/css/_extra.css                 |  15 <font color="#26A269">+++</font>
 .../src/static/js/components/list-item/Item.scss   |   5 <font color="#26A269">+</font>
 .../static/js/components/media-page/MediaPage.scss |   7 <font color="#26A269">++</font>
 static/css/_commons.css                            |   2 <font color="#26A269">+</font><font color="#C01C28">-</font>
 static/css/_extra.css                              |  15 <font color="#26A269">+++</font>
 static/css/media.css                               |   2 <font color="#26A269">+</font><font color="#C01C28">-</font>
 static/js/media.js                                 |   2 <font color="#26A269">+</font><font color="#C01C28">-</font>
 uploader/fineuploader.py                           |   3 <font color="#26A269">+</font><font color="#C01C28">-</font>
 15 files changed, 235 insertions(+), 135 deletions(-)
(mediacms.io) root@VMMediaCMS:/home/mediacms.io/mediacms# pip install -r requirements.txt -U # run pip install to update
Requirement already satisfied: Django==4.2.2 in /home/mediacms.io/lib/python3.10/site-packages (from -r requirements.txt (line 1)) (4.2.2)
Requirement already satisfied: djangorestframework==3.14.0 in /home/mediacms.io/lib/python3.10/site-packages (from -r requirements.txt (line 2)) (3.14.0)
Requirement already satisfied: django-allauth==0.54.0 in /home/mediacms.io/lib/python3.10/site-packages (from -r requirements.txt (line 3)) (0.54.0)
Requirement already satisfied: psycopg==3.1.9 in /home/mediacms.io/lib/python3.10/site-packages (from -r requirements.txt (line 4)) (3.1.9)
Requirement already satisfied: uwsgi==2.0.21 in /home/mediacms.io/lib/python3.10/site-packages (from -r requirements.txt (line 5)) (2.0.21)
Requirement already satisfied: django-redis==5.3.0 in /home/mediacms.io/lib/python3.10/site-packages (from -r requirements.txt (line 6)) (5.3.0)
Requirement already satisfied: celery==5.3.1 in /home/mediacms.io/lib/python3.10/site-packages (from -r requirements.txt (line 7)) (5.3.1)
Requirement already satisfied: drf-yasg==1.21.6 in /home/mediacms.io/lib/python3.10/site-packages (from -r requirements.txt (line 8)) (1.21.6)
Requirement already satisfied: Pillow==9.5.0 in /home/mediacms.io/lib/python3.10/site-packages (from -r requirements.txt (line 9)) (9.5.0)
Requirement already satisfied: django-imagekit==4.1.0 in /home/mediacms.io/lib/python3.10/site-packages (from -r requirements.txt (line 10)) (4.1.0)
Requirement already satisfied: markdown==3.4.3 in /home/mediacms.io/lib/python3.10/site-packages (from -r requirements.txt (line 11)) (3.4.3)
Requirement already satisfied: django-filter==23.2 in /home/mediacms.io/lib/python3.10/site-packages (from -r requirements.txt (line 12)) (23.2)
Requirement already satisfied: filetype==1.2.0 in /home/mediacms.io/lib/python3.10/site-packages (from -r requirements.txt (line 13)) (1.2.0)
Requirement already satisfied: django-mptt==0.14.0 in /home/mediacms.io/lib/python3.10/site-packages (from -r requirements.txt (line 14)) (0.14.0)
Requirement already satisfied: django-crispy-forms==1.13.0 in /home/mediacms.io/lib/python3.10/site-packages (from -r requirements.txt (line 15)) (1.13.0)
Requirement already satisfied: requests==2.31.0 in /home/mediacms.io/lib/python3.10/site-packages (from -r requirements.txt (line 16)) (2.31.0)
Requirement already satisfied: django-celery-email==3.0.0 in /home/mediacms.io/lib/python3.10/site-packages (from -r requirements.txt (line 17)) (3.0.0)
Requirement already satisfied: m3u8==3.5.0 in /home/mediacms.io/lib/python3.10/site-packages (from -r requirements.txt (line 18)) (3.5.0)
Requirement already satisfied: django-ckeditor==6.6.1 in /home/mediacms.io/lib/python3.10/site-packages (from -r requirements.txt (line 19)) (6.6.1)
Requirement already satisfied: django-debug-toolbar==4.1.0 in /home/mediacms.io/lib/python3.10/site-packages (from -r requirements.txt (line 20)) (4.1.0)
Requirement already satisfied: django-login-required-middleware==0.9.0 in /home/mediacms.io/lib/python3.10/site-packages (from -r requirements.txt (line 21)) (0.9.0)
Requirement already satisfied: asgiref&lt;4,&gt;=3.6.0 in /home/mediacms.io/lib/python3.10/site-packages (from Django==4.2.2-&gt;-r requirements.txt (line 1)) (3.8.1)
Requirement already satisfied: sqlparse&gt;=0.3.1 in /home/mediacms.io/lib/python3.10/site-packages (from Django==4.2.2-&gt;-r requirements.txt (line 1)) (0.5.1)
Requirement already satisfied: pytz in /home/mediacms.io/lib/python3.10/site-packages (from djangorestframework==3.14.0-&gt;-r requirements.txt (line 2)) (2024.2)
Requirement already satisfied: pyjwt[crypto]&gt;=1.7 in /home/mediacms.io/lib/python3.10/site-packages (from django-allauth==0.54.0-&gt;-r requirements.txt (line 3)) (2.9.0)
Requirement already satisfied: requests-oauthlib&gt;=0.3.0 in /home/mediacms.io/lib/python3.10/site-packages (from django-allauth==0.54.0-&gt;-r requirements.txt (line 3)) (2.0.0)
Requirement already satisfied: python3-openid&gt;=3.0.8 in /home/mediacms.io/lib/python3.10/site-packages (from django-allauth==0.54.0-&gt;-r requirements.txt (line 3)) (3.2.0)
Requirement already satisfied: typing-extensions&gt;=4.1 in /home/mediacms.io/lib/python3.10/site-packages (from psycopg==3.1.9-&gt;-r requirements.txt (line 4)) (4.12.2)
Requirement already satisfied: redis!=4.0.0,!=4.0.1,&gt;=3 in /home/mediacms.io/lib/python3.10/site-packages (from django-redis==5.3.0-&gt;-r requirements.txt (line 6)) (5.0.8)
Requirement already satisfied: click&lt;9.0,&gt;=8.1.2 in /home/mediacms.io/lib/python3.10/site-packages (from celery==5.3.1-&gt;-r requirements.txt (line 7)) (8.1.7)
Requirement already satisfied: tzdata&gt;=2022.7 in /home/mediacms.io/lib/python3.10/site-packages (from celery==5.3.1-&gt;-r requirements.txt (line 7)) (2024.1)
Requirement already satisfied: python-dateutil&gt;=2.8.2 in /home/mediacms.io/lib/python3.10/site-packages (from celery==5.3.1-&gt;-r requirements.txt (line 7)) (2.9.0.post0)
Requirement already satisfied: vine&lt;6.0,&gt;=5.0.0 in /home/mediacms.io/lib/python3.10/site-packages (from celery==5.3.1-&gt;-r requirements.txt (line 7)) (5.1.0)
Requirement already satisfied: billiard&lt;5.0,&gt;=4.1.0 in /home/mediacms.io/lib/python3.10/site-packages (from celery==5.3.1-&gt;-r requirements.txt (line 7)) (4.2.1)
Requirement already satisfied: click-repl&gt;=0.2.0 in /home/mediacms.io/lib/python3.10/site-packages (from celery==5.3.1-&gt;-r requirements.txt (line 7)) (0.3.0)
Requirement already satisfied: click-didyoumean&gt;=0.3.0 in /home/mediacms.io/lib/python3.10/site-packages (from celery==5.3.1-&gt;-r requirements.txt (line 7)) (0.3.1)
Requirement already satisfied: kombu&lt;6.0,&gt;=5.3.1 in /home/mediacms.io/lib/python3.10/site-packages (from celery==5.3.1-&gt;-r requirements.txt (line 7)) (5.4.2)
Requirement already satisfied: click-plugins&gt;=1.1.1 in /home/mediacms.io/lib/python3.10/site-packages (from celery==5.3.1-&gt;-r requirements.txt (line 7)) (1.1.1)
Requirement already satisfied: inflection&gt;=0.3.1 in /home/mediacms.io/lib/python3.10/site-packages (from drf-yasg==1.21.6-&gt;-r requirements.txt (line 8)) (0.5.1)
Requirement already satisfied: uritemplate&gt;=3.0.0 in /home/mediacms.io/lib/python3.10/site-packages (from drf-yasg==1.21.6-&gt;-r requirements.txt (line 8)) (4.1.1)
Requirement already satisfied: pyyaml&gt;=5.1 in /home/mediacms.io/lib/python3.10/site-packages (from drf-yasg==1.21.6-&gt;-r requirements.txt (line 8)) (6.0.2)
Requirement already satisfied: packaging&gt;=21.0 in /home/mediacms.io/lib/python3.10/site-packages (from drf-yasg==1.21.6-&gt;-r requirements.txt (line 8)) (24.1)
Requirement already satisfied: django-appconf in /home/mediacms.io/lib/python3.10/site-packages (from django-imagekit==4.1.0-&gt;-r requirements.txt (line 10)) (1.0.6)
Requirement already satisfied: pilkit&gt;=0.2.0 in /home/mediacms.io/lib/python3.10/site-packages (from django-imagekit==4.1.0-&gt;-r requirements.txt (line 10)) (3.0)
Requirement already satisfied: six in /home/mediacms.io/lib/python3.10/site-packages (from django-imagekit==4.1.0-&gt;-r requirements.txt (line 10)) (1.16.0)
Requirement already satisfied: django-js-asset in /home/mediacms.io/lib/python3.10/site-packages (from django-mptt==0.14.0-&gt;-r requirements.txt (line 14)) (2.2.0)
Requirement already satisfied: charset-normalizer&lt;4,&gt;=2 in /home/mediacms.io/lib/python3.10/site-packages (from requests==2.31.0-&gt;-r requirements.txt (line 16)) (3.3.2)
Requirement already satisfied: certifi&gt;=2017.4.17 in /home/mediacms.io/lib/python3.10/site-packages (from requests==2.31.0-&gt;-r requirements.txt (line 16)) (2024.8.30)
Requirement already satisfied: urllib3&lt;3,&gt;=1.21.1 in /home/mediacms.io/lib/python3.10/site-packages (from requests==2.31.0-&gt;-r requirements.txt (line 16)) (2.2.3)
Requirement already satisfied: idna&lt;4,&gt;=2.5 in /home/mediacms.io/lib/python3.10/site-packages (from requests==2.31.0-&gt;-r requirements.txt (line 16)) (3.10)
Requirement already satisfied: iso8601 in /home/mediacms.io/lib/python3.10/site-packages (from m3u8==3.5.0-&gt;-r requirements.txt (line 18)) (2.1.0)
Requirement already satisfied: prompt-toolkit&gt;=3.0.36 in /home/mediacms.io/lib/python3.10/site-packages (from click-repl&gt;=0.2.0-&gt;celery==5.3.1-&gt;-r requirements.txt (line 7)) (3.0.47)
Requirement already satisfied: amqp&lt;6.0.0,&gt;=5.1.1 in /home/mediacms.io/lib/python3.10/site-packages (from kombu&lt;6.0,&gt;=5.3.1-&gt;celery==5.3.1-&gt;-r requirements.txt (line 7)) (5.2.0)
Requirement already satisfied: cryptography&gt;=3.4.0 in /home/mediacms.io/lib/python3.10/site-packages (from pyjwt[crypto]&gt;=1.7-&gt;django-allauth==0.54.0-&gt;-r requirements.txt (line 3)) (43.0.1)
Requirement already satisfied: defusedxml in /home/mediacms.io/lib/python3.10/site-packages (from python3-openid&gt;=3.0.8-&gt;django-allauth==0.54.0-&gt;-r requirements.txt (line 3)) (0.7.1)
Requirement already satisfied: async-timeout&gt;=4.0.3 in /home/mediacms.io/lib/python3.10/site-packages (from redis!=4.0.0,!=4.0.1,&gt;=3-&gt;django-redis==5.3.0-&gt;-r requirements.txt (line 6)) (4.0.3)
Requirement already satisfied: oauthlib&gt;=3.0.0 in /home/mediacms.io/lib/python3.10/site-packages (from requests-oauthlib&gt;=0.3.0-&gt;django-allauth==0.54.0-&gt;-r requirements.txt (line 3)) (3.2.2)
Requirement already satisfied: cffi&gt;=1.12 in /home/mediacms.io/lib/python3.10/site-packages (from cryptography&gt;=3.4.0-&gt;pyjwt[crypto]&gt;=1.7-&gt;django-allauth==0.54.0-&gt;-r requirements.txt (line 3)) (1.17.1)
Requirement already satisfied: wcwidth in /home/mediacms.io/lib/python3.10/site-packages (from prompt-toolkit&gt;=3.0.36-&gt;click-repl&gt;=0.2.0-&gt;celery==5.3.1-&gt;-r requirements.txt (line 7)) (0.2.13)
Requirement already satisfied: pycparser in /home/mediacms.io/lib/python3.10/site-packages (from cffi&gt;=1.12-&gt;cryptography&gt;=3.4.0-&gt;pyjwt[crypto]&gt;=1.7-&gt;django-allauth==0.54.0-&gt;-r requirements.txt (line 3)) (2.22)
(mediacms.io) root@VMMediaCMS:/home/mediacms.io/mediacms# python manage.py migrate # run Django migrations
<font color="#2AA1B3"><b>Operations to perform:</b></font>
<b>  Apply all migrations: </b>account, actions, admin, auth, authtoken, contenttypes, files, sessions, sites, socialaccount, users
<font color="#2AA1B3"><b>Running migrations:</b></font>
  No migrations to apply.
<font color="#C01C28">  Your models in app(s): &apos;files&apos; have changes that are not yet reflected in a migration, and so won&apos;t be applied.</font>
<font color="#C01C28">  Run &apos;manage.py makemigrations&apos; to make new migrations, and then re-run &apos;manage.py migrate&apos; to apply them.</font>
(mediacms.io) root@VMMediaCMS:/home/mediacms.io/mediacms# python manage.py makemigrations
<font color="#2AA1B3"><b>Migrations for &apos;files&apos;:</b></font>
  <b>files/migrations/0004_alter_media_state.py</b>
    - Alter field state on media
(mediacms.io) root@VMMediaCMS:/home/mediacms.io/mediacms# python manage.py migrate # run Django migrations
<font color="#2AA1B3"><b>Operations to perform:</b></font>
<b>  Apply all migrations: </b>account, actions, admin, auth, authtoken, contenttypes, files, sessions, sites, socialaccount, users
<font color="#2AA1B3"><b>Running migrations:</b></font>
  Applying files.0004_alter_media_state...<font color="#26A269"><b> OK</b></font>
(mediacms.io) root@VMMediaCMS:/home/mediacms.io/mediacms# sudo systemctl restart mediacms celery_long celery_short # restart services
(mediacms.io) root@VMMediaCMS:/home/mediacms.io/mediacms# 
</pre>

**Reply by mgogoulos** [reply]
these logs don't show something indicative, can you do the `tail -f logs/*` , then go and upload a video, and share that output?

**Comment by mgogoulos** [comment]
@MAXimized490 this seems to be a different issue in your case, cannot debug without some additional context
